src/lib/bundle.js

```diff
@@ -0,0 +1,37 @@
+import FTLRuntimeParser from './format/ftl/entries/parser';
+import { format } from './resolver';
+
+export class Bundle {
```

Comment:
I like `MessageContext` and we can think about renaming our `L20nContext` to something else, but I'm ok with leaving the names for now.